locales/help_enu/gkhAuthors.html

```diff
@@ -41,7 +41,7 @@ <h3>Thanks!</h3>
 <li>Svyatoslav Matveev - development of rpm-package for Fedora / Rosa Linux.</li>
 <li>Alexey Klimin - testing and recommendations.</li>
 <li>Michal Novák - Czech translation.</li>
-<li>Kevin Sandal - English help content.</li>
+<li><a href="mailto:kdsandal@gmail.com?subject=GEDKeeper">Kevin Sandal</a> - English help content.</li>
```

Comment:
Generally, within the limits of decency, nobody in this section entered itself. And here you see mail of at least one person? And you are sure that you wrote all the contents of the English reference, but did not make a lot of small and not yet agreed changes?